TypeCobol/Compiler/Types/Type.cs

```diff
@@ -0,0 +1,281 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Parser;
+using TypeCobol.Compiler.Parser.Generated;
+using TypeCobol.Compiler.Symbols;
+using static TypeCobol.Compiler.Symbols.Symbol;
+
+namespace TypeCobol.Compiler.Types
+{
+    /// <summary>
+    /// A Cobol Type
+    /// </summary>
+    public class Type : ISemanticData, ICloneable
+    {
+        /// <summary>
+        /// Type tags
+        /// </summary>
+        public enum Tags
```

Comment:
What is the difference between Tags and the Type class hierarchy ?

Reply:
Tags can be used to quickly identifying from the base class Type the Kind of Type (like CodeElementType, without having to use the C# **is** operator), see poperty public Tags Tag

Reply:
Ok, can you write that in the comment of the enum ?